applications/optimization/max_clique/max_clique.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>The font/style seems messed up here..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>